jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -152,6 +184,41 @@ public void setContainerCmd(@Nullable List<String> cmd) {
     config.Cmd = cmd;
   }
 
+  public void setContainerHealthCheckTest(@Nullable List<String> test) {
```

Comment:
Just wondering if it's safe to accept null, because I see you had the precondition checks for the `DockerHealthCheck` command.